place_and_route/private/clock_tree_synthesis.bzl

```diff
@@ -47,6 +47,7 @@ def clock_tree_synthesis(ctx, open_road_info):
             sink_clustering_max_diameter = "-sink_clustering_max_diameter {}".format(ctx.attr.sink_clustering_max_diameter) if ctx.attr.sink_clustering_max_diameter else "",
         ),
         "set_propagated_clock [all_clocks]",
+        "estimate_parasitics -placement",
         "repair_clock_nets",
         "estimate_parasitics -placement",
```

Comment:
Is `estimate_parasitics -placement` still needed after `repair_clock_nets` now?

Reply:
I think yes, as `repair_clock_nets` may add extra buffer cells.